src/Sulu/Component/Content/Types/ResourceLocator/Mapper/PhpcrMapper.php

```diff
@@ -257,6 +258,8 @@ public function loadByResourceLocator($resourceLocator, $webspaceKey, $languageC
             }
         } catch (PathNotFoundException $e) {
             throw new ResourceLocatorNotFoundException(\sprintf('Path "%s" not found', $path), null, $e);
+        } catch (RepositoryException $e) {
```

Comment:
I would not catch the repository exception here.

I think it would be better to check `$path` is a validabsolutepath via the PathHelper so we even do not hit the database here:

https://github.com/phpcr/phpcr-utils/blob/6b5db3321d6f9e7ceaf3880131a089e8e2505cfc/src/PHPCR/Util/PathHelper.php#L47